numpile.py

```diff
@@ -9,18 +9,25 @@
 import pprint
 import string
 import numpy as np
-from itertools import tee, izip
 
 from textwrap import dedent
 from collections import deque, defaultdict
 
-import llvm.core as lc
-import llvm.passes as lp
-import llvm.ee as le
-from llvm.core import Module, Builder, Function, Type, Constant
+import llvmlite.llvmpy.core as lc
+import llvmlite.llvmpy.passes as lp
+#import llvm.ee as le
+from llvmlite.llvmpy.core import Module, Builder, Function, Type, Constant
 
 DEBUG = False
 
+def eager_map(f, l):
```

Comment:
Why this func, why not use list comprehensions directly (which is idiomatic Python)?